lib/dotcom_web/controllers/schedule/all_stops.ex

```diff
@@ -14,7 +16,7 @@ defmodule DotcomWeb.ScheduleController.AllStops do
   end
 
   def do_call(conn, opts) do
-    repo_fn = Keyword.get(opts, :repo_fn, &Stops.Repo.by_route/3)
+    repo_fn = Keyword.get(opts, :repo_fn, &@stops_repo.by_route/3)
```

Comment:
This could be removed and we can call the function `by_route` directly in the `get_all_stops` function

Reply:
This file is getting removed in the next PR (it's unused)